jar-connector/pom.xml

```diff
@@ -25,6 +25,12 @@
 
 		<dependency>
 			<groupId>com.sixsq.slipstream</groupId>
+			<artifactId>auth</artifactId>
+			<version>2.19-SNAPSHOT</version>
```

Comment:
Should either be removed if put into main `pom.xml` file or changed to `${project.version}`.